HHS/Head-Start-TTADP
# [TTAHUB-2795] GoalStatusChange migration for historical data

## Description of change

Pulls historical data from the audit log to populate GoalStatusChanges.

## How to test

Run the migration after importing a recent dump, check the GoalStatusChanges table and ensure it was populated.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2795


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
